Makefile

```diff
@@ -133,8 +134,11 @@ test: all
 ############# LOCAL INSTALLATION AND DOWNSTREAM PACKAGING ##############
 #
 version: VERSION mods/ra/mod.yaml mods/cnc/mod.yaml mods/d2k/mod.yaml mods/ts/mod.yaml mods/modcontent/mod.yaml mods/all/mod.yaml
-	@sh -c '. ./packaging/functions.sh; set_engine_version $(VERSION) .'
-	@sh -c '. ./packaging/functions.sh; set_mod_version $(VERSION) mods/ra/mod.yaml mods/cnc/mod.yaml mods/d2k/mod.yaml mods/ts/mod.yaml mods/modcontent/mod.yaml mods/all/mod.yaml'
+ifeq ($(GIT_VERSION), git-)
+	$(error Unable to determine version from git repository)
+endif
+	@sh -c '. ./packaging/functions.sh; set_engine_version "$(GIT_VERSION)" .'
```

Comment:
You did not modify this line, but.
This calls `set_engine_version` on the current working directory `.` I assume. Yet it runs an `awk` command on the directory which fails. 
Should this line be even there?

(Also i would have expected the shell script to be executed and not sourced `'@./packaging/function.sh --set_engine_version "$(GIT_VERSION)" .'` )

Reply:
function.sh is sourced to import the set_engine_version function, which is then called.

Reply:
I misread. The first line calls 'set_engine_version' not also 'set_mod_version'.